[2025-10-07T10:16:42.608+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:16:42.739+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=1 [queued]>
[2025-10-07T10:16:42.792+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=1 [queued]>
[2025-10-07T10:16:42.803+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:16:42.904+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:01:57.519057+00:00
[2025-10-07T10:16:43.040+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=286) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:16:43.056+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:01:57.519057+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdybx9hy5', '--map-index', '1']
[2025-10-07T10:16:43.070+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask process_batch
[2025-10-07T10:16:43.071+0000] {standard_task_runner.py:72} INFO - Started process 480 to run task
[2025-10-07T10:16:43.280+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:16:43.738+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=1 [running]> on host e3278e0f0bbc
[2025-10-07T10:16:52.132+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:01:57.519057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:01:57.519057+00:00'
[2025-10-07T10:16:52.141+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:16:52.145+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.156+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.157+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.158+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.166+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.167+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.170+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.171+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VSA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.174+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VRG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.185+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VRE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.186+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VRC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.194+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VQC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.194+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.195+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.195+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.195+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.196+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.197+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.197+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.198+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.199+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.200+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.200+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VPA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.201+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VOS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.203+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VOC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.204+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNZ: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.204+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNY: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.205+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNX: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.207+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.209+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.215+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.216+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.217+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.217+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.221+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.221+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.222+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.222+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.223+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.224+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VND: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.225+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.225+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.226+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VNA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.226+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.227+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.232+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMK: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.233+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.234+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.234+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.235+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VMC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.235+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:16:52.247+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:16:52.249+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:01:57.519057+00:00, map_index=1, execution_date=20251007T100157, start_date=20251007T101642, end_date=20251007T101652
[2025-10-07T10:16:52.548+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:16:52.897+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:16:52.912+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
